rebar.config

```diff
@@ -1,16 +1,14 @@
 %% -*- mode: Erlang; fill-column: 80; comment-column: 75; -*-
 %% Dependencies ================================================================
-{deps, [{rebar_vsn_plugin, ".*",
-         {git, "https://github.com/erlware/rebar_vsn_plugin.git",
-          {tag, "master"}}}]}.
+{deps, []}.
 
 %% Rebar Plugins ==============================================================
-{plugins, [rebar_vsn_plugin]}.
+{plugins, []}.
 
 %% Compiler Options ============================================================
 {erl_opts,
- [debug_info,
-  warnings_as_errors]}.
```

Comment:
What was your logic behind taking out w_as_errors?

Reply:
This was a leftover in testing. I put it back.